docs/subnets/subnet-development-lifecycle.md

```diff
@@ -0,0 +1,117 @@
+# What's the Subnet Development Lifecycle?
+
+As you begin your Subnet journey, it's useful to look at the lifecycle of taking a Subnet from idea
+to production.
+
+## Figure Out Your Needs
+
+The first step of planning your Subnet is determining your applications needs. What features do you
+need that the Avalanche C-Chain doesn't provide? Perhaps you want your own gas token or only want
+to allow access to KYCed customers. [When to Build on a Subnet vs. on the C-Chain](./when-to-use-subnet-vs-c-chain.md)
+can help you make the decision.
+
+### Decide What Type of Subnet You Want
+
+Once you've decided to use a Subnet, you need to decide what type of Subnet to build. This means
+choosing a virtual machine (VM) to create your Subnet with. Broadly speaking, there are three types
+of VMs to choose from:
+
+#### EVM-Based Subnets
+
+EVM-based Subnets are forks of the Avalanche C-Chain. They support Solidity smart contracts and
+standard [Ethereum APIs](/apis/avalanchego/apis/c-chain#ethereum-apis). Subnet-EVM is Ava Labs'
+implementation of an EVM-Based Subnet.
+
+Subnet-EVM is far and away the safest and most popular choice to build your Subnet with. It has the
+most mature developer tooling and receives regular updates by the Ava Labs team.
+
+#### Experimental Subnets
+
+Experimental Subnets are proof-of-concept VMs developed by Ava Labs. They include the
+[SpacesVM](spaces), [TimestampVM](create-a-vm-timestampvm), [BlobVM](create-a-vm-blobvm), and others.
```

Comment:
Do we want to mention that there are two versions of TimestampVM:
* Go: https://github.com/ava-labs/timestampvm
* Rust: https://github.com/ava-labs/timestampvm-rs